guides/common/attributes-satellite.adoc

```diff
@@ -81,6 +80,7 @@
 :freeipaserver-example-com: idm-server.example.com
 :hammer-smart-proxy: hammer capsule
 :Insights: Red{nbsp}Hat Lightspeed
+:advisorengine: {Insights} in Satellite
```

Comment:
The new IoP under development includes both Advisor and Vulnerabilities.

Does `:advisorengine:` refer to IoP in general, or Advisor specifically? Here it implies general, but the change in attributes-base.adoc implies specific.

Reply:
I don't know anymore to be honest. What should I do?

Reply:
Well, :advisorengine: is currently used like this: https://theforeman-foreman-documentation-preview-pr-4054.surge.sh/nightly/Installing_Server/index-satellite.html#installing-and-configuring-advisorengine

Is it correct or not?